app/views/monograph_catalog/_index_monograph.html.erb

```diff
@@ -25,10 +27,14 @@
         <div class="btn-toolbar" role="toolbar" aria-label="">
           <div class="btn-group" role="group" aria-label="">
             <% if @monograph_presenter.epub? %>
-                <a id="monograph-read-btn" href="<%= epub_path(@monograph_presenter.epub) %>" title="<%= t('monograph_catalog.index.read', title: @monograph_presenter.title) %>" role="button" class="btn btn-default btn-small" data-turbolinks="false"><%= t('monograph_catalog.index.read_book') %></a>
```

Comment:
Keep the original markup as an `<a ... >` but remove the `role="button"` attribute from the tag.